jdl/chevrotain.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'chevrotain' {
```

Comment:
Why is it needed? Chevrotain is written in TS, there shouldn't be an issue here

Reply:
There seems to be an issue with how chevrotain defines its types because without this TypeScript can't find the module declaration for `chevrotain`.